code/headers/mlx90640_processor.hpp

```diff
@@ -0,0 +1,121 @@
+#pragma once
+
+#include <array>
+#include <cmath>
+#include <mlx90640_i2c.hpp>
+
+namespace r2d2::thermal_camera {
+    /**
+     * MLX90640 16 bits registers
+     * INTERNAL means internal register.
+     * EE means EEPROM (Electric Erasable Programmable Read-Only)
+     * RAM means just RAM (Random Acces Memory)
+     **/
+    enum : uint16_t {
```

Comment:
Are these values used outside the mlx_processor?

Reply:
Yes the following methods use `INTERNAL_CONTROL_REGISTER` in class `mlx90640`:
  - `get_refresh_rate`
  - `set_refresh_rate`
  - `set_reading_pattern`

and in class `mlx90640_processor_c`:
  - `get_resolution_correlation`

`INTERNAL_STATUS_REGISTER` is not used by `mlx90640_processor_c`. But will maybe in the future.

and `INTERNAL_STATUS_REGISTER` in class mlx90640 are used by:
  - `frame_available`

Reply:
oke sure